### PR TITLE
PMM-10710 fix build

### PR DESCRIPTION
**Pull request by BupycHuk**
PMM-10710

Build: SUBMODULES-2768

- [ ] Links to other linked pull requests (optional).

### CODE REVIEW

**Review by atymchuk — APPROVED** [review]
Was it successful?

**Comment by BupycHuk** [comment]
@atymchuk not yet, I need the packaging team to build a new docker image for rpmbuild to make it work.

**Comment by atymchuk** [comment]
> @atymchuk not yet, I need the packaging team to build a new docker image for rpmbuild to make it work.

Packaging team? Who is that?

**Comment by BupycHuk** [comment]
Eugeny Patlan and his team :)

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # This Dockerfile used only for the API tests.
 
-FROM golang:1.18
+FROM golang:1.18.5
```
Comment (artemgavrilov) [inline Dockerfile:3]: Do we want to specify exact patch versions? That means that we should update it more often.

Comment (BupycHuk) [reply Dockerfile]: Good point, Updated to just 1.19

File: build/docker/rpmbuild/Dockerfile [hunk 2]
```diff
@@ -23,7 +23,7 @@ RUN yum update -y && \
     yum clean all && rm -rf /var/cache/yum
 
 # keep that format for easier search
-ENV GO_VERSION 1.18.4
+ENV GO_VERSION 1.19.1
```
Comment (artemgavrilov) [inline build/docker/rpmbuild/Dockerfile:26]: @BupycHuk I guess we can do the same here.

Comment (artemgavrilov) [reply build/docker/rpmbuild/Dockerfile:26]: Ok

**Review by artemgavrilov — APPROVED** [review]

**Pull request merged**